lib/hexpm/web/templates/package/_package.html.eex

```diff
@@ -1,7 +1,10 @@
 <li>
   <div class="downloads-info">
-    <span class="download-count"><%= human_number_space(@package_downloads || 0) %></span> <br>
-    downloads
+    <span class="download-count"><%= human_number_space(display_downloads(@package_downloads, @view)|| 0) %></span> <br>
+    <div class="downloads-count-wrapper">
+      <span class="downloads-count-desc"><%= display_downloads_view_title(@view) %></span><br>
+      <span class="tooltiptext"><%= display_downloads_for_opposite_views(@package_downloads, @view) %></span>
+    </div>
```

Comment:
Can you show screenshots of the UI changes?

Reply:
Added below.